account_bank_statement_import_coda/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#     This file is part of account_bank_statement_import_coda,
+#     an Odoo module.
+#
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+#
+#     account_bank_statement_import_coda is free software:
+#     you can redistribute it and/or modify it under the terms of the GNU
+#     Affero General Public License as published by the Free Software
+#     Foundation,either version 3 of the License, or (at your option) any
+#     later version.
+#
+#     account_bank_statement_import_coda is distributed
+#     in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+#     even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#     PURPOSE.  See the GNU Affero General Public License for more details.
+#
+#     You should have received a copy of the GNU Affero General Public License
+#     along with account_bank_statement_import_coda.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Import CODA Bank Statement',
+    'author': "ACSONE SA/NV,Odoo Community Association (OCA)",
+    'website': "http://www.acsone.eu",
+    'category': 'Accounting & Finance',
+    'version': '1.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'account_bank_statement_import',
+        'l10n_be_coda'
```

Comment:
If you need coda, you probably depend from l10n_be for your business. But if you install l10n_be, l10n_be_coda will be installed as a dependency of l10n_be. By depending of l10n_be_coda, we can safely share the same model, avoid to redefine the same field in the account_bank_statement view and hide the menu item that leads to 'old style' import wizard provided by l10n_be_coda.